<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lesson_manage_page</class>
 <widget class="QWidget" name="lesson_manage_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>课程信息管理</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icon_rect.png</normaloff>:/icon_rect.png</iconset>
  </property>
  <widget class="QTreeWidget" name="content">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>651</width>
     <height>480</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string notr="true">1</string>
    </property>
   </column>
  </widget>
  <widget class="QWidget" name="top_widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>80</height>
    </rect>
   </property>
   <widget class="QPushButton" name="turn2student_manage">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>200</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="toolTip">
     <string>转到课程信息管理界面</string>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>转到学生信息</string>
    </property>
   </widget>
   <widget class="QLabel" name="topic">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>400</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>课程信息管理</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="upper_right" native="true">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>0</y>
      <width>200</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="username_main">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>username</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="btn_exit">
       <property name="text">
        <string>退出</string>
       </property>
       <property name="icon">
        <iconset theme="application-exit">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>108</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>更改信息</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="add_score_btn">
      <property name="text">
       <string>新建成绩</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="del_score_btn">
      <property name="text">
       <string>删除成绩</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="chag_score_btn">
      <property name="text">
       <string>修改成绩</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>95</width>
     <height>124</height>
    </rect>
   </property>
   <property name="title">
    <string>课程总览</string>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="total_stu">
      <property name="text">
       <string>总人数：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="pass_rate">
      <property name="text">
       <string>及格率：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="average">
      <property name="text">
       <string>平均分：</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>661</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLineEdit" name="search_bar_lesson"/>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="find_stu">
      <property name="text">
       <string>查找</string>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
